bin/composer-script/Script.php

```diff
@@ -30,9 +35,42 @@ protected static function getProjectFolder(Event $event): string
     /**
      * Execute a command in the CLI, as a separate process.
      */
-    protected static function run(string $command): void
+    protected static function run(string $command): int
     {
-        $process = new Process($command);
-        $process->run();
+        // Depending on the context, we're using either Symfony/Process 2.8.52 (bundled with composer 2.1.x)
+        // or Symfony/Process 5.3.x (if we're using our own). The signature of the Constructor changed
+        // from: `public function __construct(string $commandline, …)`
+        // to:   `public function __construct(array $command, …)`
+        // We'll have to attempt one, and otherwise fall back to the other.
+
+        try {
+            $process = new Process($command);
+        } catch (\TypeError $e) {
+            $process = new Process([$command]);
+        }
```

Comment:
Safer would be:
```suggestion
        $process = Process::fromShellCommandline($command);
```

Reply:
I will try that out.. If that works, that'll be a lot less fugly! 

Thanks for the suggestion, @stloyd!

Reply:
Alas, it doesn't seem to work in the context of `composer create-project`, because it seems that method isn't present yet in `symfony/process` 2.8.x, as used by Composer. 

It breaks with this error: 

```bash
…
…
120 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

> Bolt\ComposerScripts\ProjectEventHandler::postUpdate

 ! [NOTE] Running composer "post-update-cmd" scripts

PHP Fatal error:  Uncaught Error: Call to undefined method Symfony\Component\Process\Process::fromShellCommandline() in /Users/bob/Sites/Bolt/playground/vendor/bolt/core/bin/composer-script/Script.php:39
Stack trace:
#0 /Users/bob/Sites/Bolt/playground/vendor/bolt/core/bin/composer-script/PostUpdateScript.php(13): Bolt\ComposerScripts\Script::run('php vendor/bobd...')
#1 /Users/bob/Sites/Bolt/playground/vendor/bolt/core/bin/composer-script/ProjectEventHandler.php(44): Bolt\ComposerScripts\PostUpdateScript::execute()
#2 phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(377): Bolt\ComposerScripts\ProjectEventHandler::postUpdate(Object(Composer\Script\Event))
#3 phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(236): Composer\EventDispatcher\EventDispatcher->executeEventPhpScript('Bolt\\ComposerSc...', 'postUpdate', Object(Composer\Script\Event))
#4 phar:///usr/local/bin/composer/src/Composer/EventDispatcher/EventDispatcher.php(117): Composer\EventDispatcher\EventDispatc in /Users/bob/Sites/Bolt/playground/vendor/bolt/core/bin/composer-script/Script.php on line 39
```